src/main/target/REVO/target.h

```diff
@@ -222,16 +221,21 @@
 #define SPI3_MISO_PIN           PC11
 #define SPI3_MOSI_PIN           PC12
 
+#define USE_I2C
 #if defined(AIRBOTF4) || defined(AIRBOTF4SD)
 // On AIRBOTF4 and AIRBOTF4SD, I2C2 and I2C3 are configurable
-#define USE_I2C
 #define USE_I2C_DEVICE_2
 #define I2C2_SCL                NONE // PB10, shared with UART3TX
 #define I2C2_SDA                NONE // PB11, shared with UART3RX
 #define USE_I2C_DEVICE_3
 #define I2C3_SCL                NONE // PA8, PWM6
 #define I2C3_SDA                NONE // PC9, CH6
 #define I2C_DEVICE              (I2CDEV_2)
+#else
+#define USE_I2C_DEVICE_1
+#define I2C_DEVICE              (I2CDEV_1)
+#define I2C1_SCL                PB8
+#define I2C2_SDA                PB9
```

Comment:
@jflyper There's small typo:
Change
#define I2C2_SDA PB9
into
#define I2C1_SDA PB9
With this baro and mag work again on a REVO